code/controllers/configuration.dm

```diff
@@ -96,9 +96,6 @@
 	var/health_threshold_crit = 0
 	var/health_threshold_dead = -100
 
-	var/organ_health_multiplier = 1
-	var/organ_regeneration_multiplier = 1

```

Comment:
We've had external wound regeneration disabled for 3 years now; on top of which, with how we changed `take_overall_damage` and `heal_overall_damage` to actually do as the name implies....well....you'd be able to heal from even 70+ damage wounds.